Dockerfile

```diff
@@ -1,34 +1,59 @@
 # https://docs.docker.com/reference/builder/
 # NOTE dockerignore is ignored https://github.com/docker/docker/issues/9455
 
-FROM ensime/ensime:v1.x
-MAINTAINER Sam Halliday, sam.halliday@gmail.com
+FROM debian:jessie
```

Comment:
do you not want to base off one of the existing ensime images? Basing off `ensime/ensime:v2.x` would seem sensible given that you want Java 7.

BTW, we'll have the coursier cache in `v1.x` soon when https://github.com/ensime/ensime-server/pull/1312 is unblocked.